src/components/LuxAutocompleteInput.vue

```diff
@@ -28,7 +28,7 @@
           v-else
           v-for="(result, i) in results"
           :key="i"
-          @click="setResult(result, emitSelectedId)"
+          @mousedown="onClickResult(result)"
```

Comment:
We should deal with enter the same way we are dealing with click.   I'm guessing we would have to check key down and then which key, but then you would not need to keep track of arrow keys, just pass the item that got the keydown.

Reply:
I think enter still needs slightly different handling, because the actual browser focus never leaves the `<input>`, even when using the arrow keys to move up and down (note that the `@keydown.enter` event binding is on the `<input>`.